README.md

```diff
@@ -28,16 +28,6 @@ Contact us with a bug or suggestion by making an issue.
 
 Checkout [who built this](https://github.com/CobeyH/SENG-480A/wiki/Team-Contributions) application on our [wiki](https://github.com/CobeyH/SENG-480A/wiki)
```

Comment:
This is still linking to our old repository. We should probably set this to link to the Tandem Repo.